src/components/Timeline/_timeline.scss

```diff
@@ -9,11 +9,10 @@
 // .cc-timeline {}
 
 .cc-timeline__title {
-  @include heading-divider;
+  @include heading-fancy;
```

Comment:
I am not sure this, I couldn't find any example of it on the page...

Reply:
https://wellcome.org/what-we-do/our-work/wellcome-photography-prize/2021

Reply:
great, thanks!